CONTRIBUTING.md

```diff
@@ -0,0 +1,80 @@
+# GlusterFS project Contribution guidelines
+
+## Development Workflow
+
+0. Fork Repository
+   - Visit [GlusterFS repository](https://github.com/gluster/glusterfs) and click 'Fork'.
```

Comment:
Direct link to Fork https://github.com/gluster/glusterfs/fork

Reply:
Done